spot_driver/launch/rviz.launch.py

```diff
@@ -0,0 +1,82 @@
+# Copyright [2023] Boston Dynamics AI Institute, Inc.
```

Comment:
The Quality of Service needs to be set to "best effort" in Rviz if you want to visualize the PointCloud2 under `Reliability Policy`. Is there a way to set that automatically in this config?

Reply:
There is a way to have that default to the right setting, but I think you need to provide that in the config YAML file. Here's the full entry for a PointCloud display, for example. We set `Topic -> Reliability Policy` to `Best Effort` instead of `Reliable`.

```
    - Alpha: 1
      Autocompute Intensity Bounds: true
      Autocompute Value Bounds:
        Max Value: 3.0432052612304688
        Min Value: -0.07985210418701172
        Value: true
      Axis: Z
      Channel Name: intensity
      Class: rviz_default_plugins/PointCloud2
      Color: 255; 255; 255
      Color Transformer: AxisColor
      Decay Time: 0
      Enabled: true
      Invert Rainbow: false
      Max Color: 255; 255; 255
      Max Intensity: 4096
      Min Color: 0; 0; 0
      Min Intensity: 0
      Name: PointCloud2
      Position Transformer: XYZ
      Selectable: true
      Size (Pixels): 3
      Size (m): 0.009999999776482582
      Style: Flat Squares
      Topic:
        Depth: 5
        Durability Policy: Volatile
        History Policy: Keep Last
        Reliability Policy: Best Effort
        Value: /points_back
      Use Fixed Frame: true
      Use rainbow: true
      Value: true
```